web/ui_utils.js

```diff
@@ -227,8 +227,8 @@ function removeNullCharacters(str, replaceInvisible = false) {
  * @returns {number} Index of the first array element to pass the test,
  *                   or |items.length| if no such element exists.
  */
-function binarySearchFirstItem(items, condition) {
-  let minIndex = 0;
+function binarySearchFirstItem(items, condition, start = 0) {
+  let minIndex = start;
```

Comment:
These changes probably deserve extending the existing unit-tests to also cover the new functionality; see https://github.com/mozilla/pdf.js/blob/master/test/unit/ui_utils_spec.js